src/main/java/io/tackle/pathfinder/services/AssessmentSvc.java

```diff
@@ -316,6 +317,20 @@ public void deleteAssessment(@NotNull Long assessmentId) {
         if (!deleted) throw new BadRequestException();
     }
 
+    @Transactional
+    public void bulkDeleteAssessments(@NotNull List <Long> applicationIds) {
+        applicationIds.forEach(applicationId -> {
+            PanacheQuery<Assessment> query = Assessment.find("application_id", applicationId);
+            List<Assessment> assessments = query.stream().collect(Collectors.toList());
+            assessments.forEach(assessment -> {
+                deleteAssessment(assessment.id);
+            });
+
+        });
+        //Assessment.delete("application_id in ?1",applicationIds);
```

Comment:
AFAIU, this is the first approach you tried which wasn't working due to an issue.
It's fine for me to keep it commented so that in the future, if for whatever reason we try it again, we stop immediately but it would be great for add a comment saying something like `This approach is not working due to <link to Quarkus issue>`